### PR TITLE
Create submit

### PR DESCRIPTION
**Pull request by Schleppguy**

### CODE REVIEW

File: server/controllers/events.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const models = require('../../db/models');
+
+module.exports.create = (req, res) => {
+  console.log(req.body);
+  models.Event.forge({title: req.body.eventName, creator_id: req.session.passport.user})
```
Comment (amatranga) [inline server/controllers/events.js]: should creator_id be camel case?
creatorId?

File: server/controllers/events.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  //   .then(result => {
+  //     res.status(201).send(result.omit('password'));
+  //   })
+  //   .catch(err => {
+  //     if (err.constraint === 'users_username_unique') {
+  //       return res.status(403);
+  //     }
+  //     res.status(500).send(err);
+  //   });
+};
```
Comment (amatranga) [inline server/controllers/events.js]: Is the commented out code on lines 21-31 needed?

File: server/controllers/events.js (hunk 1 above)
Comment (amatranga) [inline server/controllers/events.js:5]: Should creator_id be camel case?
creatorId?

File: server/controllers/events.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  //   .save()
+  //   .then(result => {
+  //     res.status(201).send(result.omit('password'));
+  //   })
+  //   .catch(err => {
+  //     if (err.constraint === 'users_username_unique') {
+  //       return res.status(403);
+  //     }
+  //     res.status(500).send(err);
+  //   });
```
Comment (amatranga) [inline server/controllers/events.js:31]: Is the commented out code on lines 21-31 needed?

**Review by amatranga — CHANGES_REQUESTED** [review]

File: server/controllers/events.js (hunk 1 above)
Comment (Schleppguy) [reply server/controllers/events.js:5]: No, it refers to the creator_id column in the database. It has to be creator_id.

File: server/controllers/events.js (hunk 3 above)
Comment (Schleppguy) [reply server/controllers/events.js:31]: No, but that section is a work in progress. It'll be gone on my next PR. Don't usually like pushing up commented out code, but I wanted to get the current state up there so folks aren't waiting on me.

Comment (epan) [inline server/controllers/events.js]: ❤️

**Pull request merged**